.github/release.yml

```diff
@@ -0,0 +1,36 @@
+# .github/release.yml
+
+changelog:
+  exclude:
+    labels:
+      - release-note/ignore-for-release
+    authors:
+      - octocat
+  categories:
+    - title: Exciting New Features 🎉
+      labels:
+        - release-note/new-feature
+    - title: Enhancement 🚀
+      labels:
+        - release-note/enhancement
+    - title: Component updates ⬆️
+      labels:
+        - release-note/update
+    - title: Docs update 🗄️
+      labels:
+        - release-note/docs
+    - title: Community update 🧑🏻‍🤝‍🧑🏾
```

Comment:
I'd like to make sure i understand can you provide an example of what type of PR would get this label? 

And how, does a maintainer/community member know when to apply this label?

Reply:
We will add descriptions in the contributors guide in which cases which labels to be used, but I think the labels are quite self explanatory, the docs and community labels are in case we wanna do some cross repo, to include changes from community and website repos, which BTW are something else that we need to fix (future ISSUE) :)